Packs/Redmine/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### Redmine
+
+- Updated the Docker image to: *demisto/python3:3.10.14.92207*.
+- Fixed an issue with the tracker_id argument to support custom tracker IDs.
```

Comment:
maybe lets add that now you need to provide the ID?

Reply:
added